addons/web/static/src/core/utils/timing.js

```diff
@@ -33,36 +33,49 @@ export function batched(callback, synchronize = () => Promise.resolve()) {
  * @param {number | "animationFrame"} delay how long should elapse before the function
  *      is called. If 'animationFrame' is given instead of a number, 'requestAnimationFrame'
  *      will be used instead of 'setTimeout'.
- * @param {boolean} [immediate=false] whether the function should be called on
- *      the leading edge instead of the trailing edge.
+ * @param {boolean} [options] if true, equivalent to exclusive leading. If false, equivalent to exclusive trailing.
+ * @param {object} [options]
+ * @param {boolean} [options.leading=false] whether the function should be invoked at the leading edge of the timeout
+ * @param {boolean} [options.trailing=true] whether the function should be invoked at the trailing edge of the timeout,
+ *      note that if `leading` is also true, it will only be invoked at the trailing edge if the debounced function was
+ *      called at least once more during the wait time.
```

Comment:
The note should be in the function explanation block above, not in follow-up of docstring of the params.